ci/glci/model.py

```diff
@@ -465,9 +473,26 @@ class AzurePublishCfg:
     notification_recipients: ()
 
 
+@dataclasses.dataclass(frozen=True)
+class OpenstackEnviroment:
+    auth_url: str
+    domain: str
+    region: str
+    project_name: str
+    username: str
+    password: str
+
+
+@dataclasses.dataclass(frozen=True)
+class OpenstackPublishCfg:
+    openrc: OpenstackEnviroment
+    properties: dict
```

Comment:
while this technically works, of course, adding dataclasses also for the `properties` could help to improve readability / maintainability

Reply:
Hmm the issue here we need the properties dict to pass image properties which can be different depending on the openstack environment. As everything within the ci comes from the secret server we should maybe also retrieve them from there. WDYT?

Reply:
The information are now retrieved from secret server.